### PR TITLE
Add a banner linking out to KZG ceremony

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Adds a dismissable banner calling out the KZG ceremony

### CODE REVIEW

File: src/components/Banners/DismissableBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  const onClose = () => {
+    localStorage.setItem(storageKey, "true")
+    setShow(false)
+  }
+
+  return (
+    <BannerNotification shouldShow={show}>
+      <Center paddingRight={8} flexWrap="wrap">
+        <Emoji marginRight={4} flexShrink={0} text=":tada:" w={4} h={4} />
```
Comment (wackerow) [inline src/components/Banners/DismissableBanner.tsx:31]: @corwintines 
image
Getting a little buggy layout here

**Comment by nloureiro** [comment]
@corwintines @wackerow, let me know what you think of this:

1- the warning should be on top of the website because it interferes with the normal flow of the content and has different behavior 
Screen Shot 2023-02-14 03 21 22 PM

2. and how about we lower the visual relevance? The primary color might be too strong

Screen Shot 2023-02-14 03 23 44 PM


These are ideas, but probably I need to revisit this on the design system to set a clear rule on how to use it and how relevant it can be...

**Comment by wackerow** [comment]
Certainly not against trying a new design here... will note though that this banner is time sensitive. I'd prefer to minimize the blockers on this PR to get this live asap. I can peak at the emoji alignment and try to patch.

**Review by wackerow — APPROVED** [review]

**Comment by corwintines** [comment]
Id like to merge this for now, and revisit the design given the time sensitivity. @nloureiro we can open a separate PR for styles I think.

**Comment by nloureiro** [comment]
@corwintines @wackerow did you merge it? found this bug with the closing alert

https://user-images.githubusercontent.com/1120748/219365799-ca4574ab-5f98-40a6-90ac-9782616392c0.mov

**Pull request merged**